docs/guides/configuration.md

```diff
@@ -151,6 +151,19 @@ ReactOnRails.configure do |config|
   config.server_renderer_pool_size = 1 # increase if you're on JRuby
   config.server_renderer_timeout = 20 # seconds
 
+  ################################################################################
+  ################################################################################
+  # FILE SYSTEM BASED COMPONENT REGISTRY
+  ################################################################################
+  # components_directory is the directory that is used to automatically detect and register components for the usage on
+  # the rails view. default is nil, you can enable the feature by updating it in the next line.
+  config.components_directory = "ror_components"
```

Comment:
What happens if ShakaPacker is less than v6.5?

Is the crash obscure? or clear?

Let's have a clear error message if this value is set, but the version is lower than 6.5 or ShakaPacker is not available.